frontend/src/app/account/history/history.model.ts

```diff
@@ -0,0 +1,40 @@
+export interface MatchingHistory {
+    id: string;
+    collaborator: string; // collaborator username
```

Comment:
Think it would be better to just use `collaborator_username` if you think it is confusing to put collaborator haha

Reply:
Everything being one word looks nicer :p